packages/camlidl/camlidl.1.05/opam

```diff
@@ -24,14 +27,14 @@ generation of C stub code required for the Caml/C interface, based on
 an MIDL specification. Also provides some support for Microsoft's COM
 software components."""
 depends: [
-  "ocaml" {< "5.0"}
+  "ocaml" {< "4.14"}
 ]
+available: [ os != "win32" & os != "cygwin" ]
 url {
-  src:
-    "http://caml.inria.fr/pub/old_caml_site/distrib/bazar-ocaml/camlidl-1.05.tar.gz"
+  src: "https://github.com/xavierleroy/camlidl/archive/camlidl105.tar.gz"
```

Comment:
Is it possible to get this file from one of the caches and avoid changing checksums?

https://github.com/ocaml/opam-repository/wiki/FAQ#how-to-find-lost-archives-of-packages

If not, we should probably mark this as unavailable. Changing checksums is no longer allowed as a policy